bin/fuel-core/src/cli.rs

```diff
@@ -55,7 +57,7 @@ fn init_environment() -> Option<PathBuf> {
     None
 }
 
-pub async fn init_logging() -> anyhow::Result<()> {
+pub fn init_logging() {
```

Comment:
Why not return the errors rather than `expect`ing them?

I realize that it's not the end of the world if there is a panic right on startup, but curious since it's there.

Reply:
Can't speak for the author but the expect messages seem more straightforward than e.g. printing `ParseBoolError` at some point.